addons/api/addon/generated/models/storage-bucket.js

```diff
@@ -73,12 +73,6 @@ export default class GeneratedStorageBucketModel extends BaseModel {
   })
   region;
 
-  @attr('string', {
-    readOnly: true,
-    isNestedAttribute: true,
-  })
-  secrets_hmac;
```

Comment:
Do we not get a `secrets_hmac` back anymore? Why was this deleted?

Reply:
Oops no this is still needed